### PR TITLE
Redirect and Pass UTM data to Elevate

### PR DESCRIPTION
**Pull request by voduyemi**
We added the ability to redirect users on click of a bell notification

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/classes/MockAudienceImpl.cls-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>51.0</apiVersion>
```
Comment (psadie-sfdo) [inline force-app/main/default/classes/MockAudienceImpl.cls-meta.xml:3]:
```suggestion
    <apiVersion>52.0</apiVersion>
```

File: force-app/main/default/classes/OrganizationSelector.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>51.0</apiVersion>
```
Comment (psadie-sfdo) [inline force-app/main/default/classes/OrganizationSelector.cls-meta.xml:3]:
```suggestion
    <apiVersion>52.0</apiVersion>
```

File: force-app/main/default/classes/USNonElevateCustomers_TEST.cls-meta.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>51.0</apiVersion>
```
Comment (psadie-sfdo) [inline force-app/main/default/classes/USNonElevateCustomers_TEST.cls-meta.xml:3]:
```suggestion
    <apiVersion>52.0</apiVersion>
```

**Review by psadie-sfdo — COMMENTED** [review]
A few minor non-blocking requests and a question.

Functional changes are looking good! Gonna run through tests now.

File: force-app/main/default/classes/MockAudienceImpl.cls [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @description Mock Audience Implementation class for Notification Framework
+*/
+
```
[the remark this answers is not in the document]
Comment (voduyemi) [reply force-app/main/default/classes/MockAudienceImpl.cls:35]: I'm not sure actually, I'll add it

**Pull request merged**